rust/operator-binary/src/catalog/commons.rs

```diff
@@ -93,22 +95,53 @@ impl ExtendCatalogConfig for S3ConnectionInlineOrReference {
             .await
             .context(ConfigureS3Snafu)?;
 
-        catalog_config.add_property("hive.s3.endpoint", s3.endpoint().context(ConfigureS3Snafu)?);
-        catalog_config.add_property(
-            "hive.s3.path-style-access",
-            (s3.access_style == S3AccessStyle::Path).to_string(),
-        );
-
         let (volumes, mounts) = s3.volumes_and_mounts().context(ConfigureS3Snafu)?;
         catalog_config.volumes.extend(volumes);
         catalog_config.volume_mounts.extend(mounts);
 
+        if trino_version >= 469 {
+            // Since Trino 469, S3 native support has to be explicitly enabled
+            // unless using Legacy S3 support (which has been removed in 470).
```

Comment:
It's still in there AFAIK
```suggestion
            // unless using Legacy S3 support (which has been deprecated in 470).
```